vpr/src/device/rr_graph_builder_view.h

```diff
@@ -0,0 +1,53 @@
+#ifndef _RR_GRAPH_BUILDER_VIEW_
+#define _RR_GRAPH_BUILDER_VIEW_
+
+#include <exception>
```

Comment:
why do you include `<exception>` ? I don't see that needed anywhere below.

Reply:
Agree. Removed all the unused headers.